stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/MessagesActivity.kt

```diff
@@ -139,26 +142,45 @@ class MessagesActivity : AppCompatActivity() {
                 )
             }
 
-            if (selectedMessage != null) {
-                SelectedMessageMenu(
-                    modifier = Modifier
-                        .align(Alignment.Center)
-                        .padding(horizontal = 20.dp)
-                        .wrapContentSize(),
-                    shape = ChatTheme.shapes.attachment,
-                    messageOptions = defaultMessageOptionsState(
-                        selectedMessage = selectedMessage,
+            if (selectedMessageState != null) {
+                val selectedMessage = selectedMessageState.message
+                if (selectedMessageState is SelectedMessageOptionsState) {
+                    SelectedMessageMenu(
+                        modifier = Modifier
+                            .align(Alignment.Center)
+                            .padding(horizontal = 20.dp)
+                            .wrapContentSize(),
+                        shape = ChatTheme.shapes.attachment,
+                        messageOptions = defaultMessageOptionsState(
+                            selectedMessage = selectedMessage,
+                            currentUser = user,
+                            isInThread = listViewModel.isInThread
+                        ),
+                        message = selectedMessage,
+                        onMessageAction = { action ->
+                            composerViewModel.performMessageAction(action)
+                            listViewModel.performMessageAction(action)
+                            listViewModel.removeOverlay()
```

Comment:
I think calling `listViewModel.removeOverlay()` is redundant as the removal of the overlay is done automatically by the `MessageListViewModel` [here](https://github.com/GetStream/stream-chat-android/blob/6e1111dec43b4d89bd849fb71cb8f42bfcdf172d/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt#L513) inside `performMessageAction()`

Removing the call still resulted in the same functionality on my end, please check if it's the same for you.

Reply:
Agreed - if we can remove the extra calls (my bad for that), let's do that!